pyproject.toml

```diff
@@ -2,16 +2,44 @@
 build-backend = "poetry.core.masonry.api"
 requires = ["poetry-core"]
 
+[project]
+
+[project.urls]
+Repository = "https://github.com/CORDEX-be2/ValEnsPy"
+Issues = "https://github.com/CORDEX-be2/ValEnsPy/issues"
+Documentation = "https://cordex-be2.github.io/ValEnsPy/"
```

Comment:
Sidenote:

Be aware that you point to the documentation of the main branch. So if you use a valenspy version, which is not the latest, then the documentation always points to the latest version. 

In an early stage, this is not that problematic but when more people are using valenspy, i think it is best to make the documentation for all versions. This can easily be done by using readthedocs instead of github_pages for hosting your documentation.

Reply:
Yes indeed - something to look at at a later stage. Currently we will keep this as is.